index.html

```diff
@@ -793,6 +793,26 @@ <h2 id="constrainable-properties">Constrainable Properties for Captured
           </tr>
 
 
+          <tr id="def-constraint-deviceId">
+            <td><dfn class="export">deviceId</dfn>
+            </td>
+
+            <td>{{DOMString}}</td>
+
+            <td>
+              The identifier of the surface being captured.
+
```

Comment:
Judging from the preview, to me I find it incredibly hard to read given the current line breaks. Also the following bullet is hard to read. Concatenate sentences?

Reply:
It seems there is a rendering bug somewhere: the preview is not showing the black rectangle around the block like the main spec does (see https://w3c.github.io/mediacapture-screen-share/#dfn-displaysurface). 

The current separation mirrors what has been done for `displaySurface`.
That said `displaySurface` is the outlier here, all other constraints concatenate sequences.
I am fine concatenating.